src/modules/moveFiles.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { readFile } = require('./readFile');
+const { writeFile } = require('./writeFile');
+const { deleteFile } = require('./deleteFile');
+const { renameFile } = require('./renameFile');
+
+const moveFiles = (
+  command,
+  fileToMove,
+  fileToMovePath,
+  destinationToMove
+) => {
+  if (command === 'mv') {
+    const data = readFile(fileToMovePath);
+
+    if (destinationToMove.endsWith('.txt')) {
+      renameFile(fileToMovePath, destinationToMove);
+    } else if (destinationToMove.endsWith('/')) {
+      writeFile(destinationToMove + fileToMove, data);
+
+      deleteFile(fileToMovePath);
+    } else {
+      const status = writeFile(destinationToMove + '/' + fileToMove, data);
+
+      if (status === -1) {
+        writeFile(destinationToMove + '.txt', data);
+      }
+
+      deleteFile(fileToMovePath);
+    }
+  } else {
+    global.console.log('Wrong command - for moving use "mv"');
+  }
```

Comment:
The current implementation of the moveFiles function is a bit convoluted. You can simplify the code structure by adding try-catch blocks for this part, and replace some strings:
```
if (command !== 'mv') {
       throw new Error('Wrong command - for moving use "mv"');
 }
 ```